test/test_c10d.py

```diff
@@ -1115,6 +1125,15 @@ def forward(self, x):
 
 class DistributedDataParallelTest(MultiProcessTestCase):
 
+    def tearDown(self):
+        # DistributedDataParallel test doesn't seem to call FileStore destructor
+        # TODO: investigate this test and the test is known to have issues
+        # Use this hack to remove files for that test
```

Comment:
Any idea why this happens?

Reply:
No, and making it a TODO. I wouldn't worry about it for now since my own DDP full ImageNet run works fine and the file is removed.